governance/votes/Executive vote - September 13, 2023.md

```diff
@@ -0,0 +1,205 @@
+
+Phoenix Labs has proposed increasing the Spark Protocol D3M [Maximum Debt Ceiling](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) parameter as SparkLend's DAI borrow market is maxed out and requires capacity for more users. Phoenix Labs has also proposed increasing the D3M [Ceiling Increase Cooldown](https://manual.makerdao.com/module-index/module-dciam#ceiling-increase-cooldown-ttl) parameter to reduce exposure to Maker Core should a time-sensitive shutdown be required. If this executive proposal passes, the following actions will occur:
+
+- The Spark Protocol D3M (DIRECT-SPARK-DAI) Maximum Debt Ceiling will be increased from 200 million DAI to **400 million DAI.**
+- The Spark Protocol D3M (DIRECT-SPARK-DAI) Ceiling Increase Cooldown will be increased from 8 hours to **12 hours.**
+
+### Fortunafi (RWA005-A) Vault - Initiate Offboarding
+
+*Authorization*: [Stability Scope Artifact](https://mips.makerdao.com/mips/details/MIP104#14-4-legacy-legal-recourse-assets); [Governance Poll 2023-08-28](https://vote.makerdao.com/polling/Qmcb1c9x)\
+*Reasoning*: [Forum post](http://forum.makerdao.com/t/request-to-poll-offboarding-legacy-legal-recourse-assets/21582) "Offboarding Legacy Legal Recourse Assets"\
+*Authorizing party*:  [Stability Scope Facilitator](http://forum.makerdao.com/t/request-to-poll-offboarding-legacy-legal-recourse-assets/21582/8)
+
+The Stability Scope Artifact requires that legacy Legal Recourse Assets be offboarded prior to the launch of NewChain. If this executive proposal passes, the Fortunafi vault Debt Ceiling will be decreased from 15 million DAI to **zero DAI**.
+
+### DAO Resolution for HV Bank (RWA009-A) Vault
+
+*Authorization*: [Stability Scope Artifact](https://mips.makerdao.com/mips/details/MIP104#14-4-legacy-legal-recourse-assets); [Governance Poll 2023-08-28](https://vote.makerdao.com/polling/QmNgKzcG)\
+*Reasoning*: [Forum post](http://forum.makerdao.com/t/request-to-poll-offboarding-legacy-legal-recourse-assets/21582) "Offboarding Legacy Legal Recourse Assets"\
+*Authorizing party*:  [Stability Scope Facilitator](http://forum.makerdao.com/t/request-to-poll-offboarding-legacy-legal-recourse-assets/21582/8)
+
+The Stability Scope Artifact requires that legacy Legal Recourse Assets be offboarded prior to the launch of NewChain. The Stability Scope Facilitator seeks approval of a DAO Resolution that will instruct the RWA Foundation to terminate future purchases in HV Bank’s RWA Master Participation Trust and return excess available cash to the DAO. 
+
+If this executive proposal passes, the DAO Resolution with the IPFS hash [QmXU2TwsRpVevGY74NVFbD9bKwtsw1mSuSce7My1zinD9m](https://gateway.pinata.cloud/ipfs/QmXU2TwsRpVevGY74NVFbD9bKwtsw1mSuSce7My1zinD9m) will be approved.
+
+For reference, a raw version of the text may be found below:
```

Comment:
Given that you've already included a link to the DAO resolution, the raw text might be redundant and could make the document harder to digest. Could we consider keeping just the link?

Reply:
Good point. If we can make the Exec more brief without losing any vital info, we should. I'll remove it then. Thanks VW.